.pre-commit-config.yaml

```diff
@@ -360,3 +360,9 @@ repos:
         entry: "./scripts/ci/pre_commit_bat_tests.sh"
         files: ^breeze$|^breeze-complete$|\.sh$|\.bash$
         pass_filenames: false
+      - id: stylelint
+        name: stylelint
+        entry: "stylelint"
+        language: node
+        files: ^airflow/www/.*\.(css|scss|sass)$
+        args: []
```

Comment:
I think it might be safe to remove this `args` if it is just empty

Reply:
Ah, you're right. I had previously removed it but still had `pass_filenames` in there and it wasn't working. Just removed it.